src/OpenLoco/src/Vehicles/Routing.cpp

```diff
@@ -660,17 +677,17 @@ namespace OpenLoco::Vehicles
     }
 
     constexpr size_t kSignalHashMapSize = 0x400;
+    constexpr size_t kTrackModHashMapSize = 0x1000;
 
-    // 0x004A2E46
+    // 0x004A2E46 & 0x004A2DE4
```

Comment:
Out of curiosity, does @ZehMatt's function mapping script pick up on these double addresses?

Reply:
I don't think it does

Reply:
I think I read all addresses but I'll revisit this.

Reply:
could you just put the second address on a second comment line?
eg
// 0x004A2E46
// 0x004A2DE4

Reply:
That still wouldn't be picked up by the function mapping script. I'm pretty sure it only does one address to one function. I think the function names would need to be unique so it would need modification to handle the & or the second line version. Personally i feel the & is more natural.

Reply:
Yeah, I'd prefer the single line as well. Pretty sure I've used `0x...., 0x....` myself elsewhere, too.